lib/rules/indent.js

```diff
@@ -1094,6 +1094,17 @@ module.exports = {
                     parameterParens.add(closingParen);
                     addElementListIndent(node.params, openingParen, closingParen, options.FunctionExpression.parameters);
                 }
+
+                if (firstToken.type === "Identifier" && firstToken.value === "async") {
+                    const openingParen = sourceCode.getTokenAfter(firstToken);
```

Comment:
What will happen if the first token after `async` isn't an opening paren, as in `async a => {}`

Reply:
Thanks for the input.

I added a check and tests for this. 

Let me know if there is anything else.